botpress/botpress
# Head of team Bugpress

## Description

Clients were not able to access the two different google-speech config files in the code editor (local and global). I basically pushed a fix to basically recognize which json file is local and which one is global. This will allow the user to click on the one they want to work on, creating a differentiation (since the name is the same). 

Fixes botpress/v12#1483 

## Type of change


## How has this been tested?

1. Enable the Google speech module, then go on the one of your bots. 
2. Go to the code -editor and scroll down to "module configurations". 
3. Make a change in one of the relative config files and save. You should be able to see the changes made to the specific file you accessed. You should also be able to click on both json files and get two different tabs instead of one. 

**Test configuration**:
* BP version: 12.26.6
* Node version: 12.18.2 
* OS: MacOS
* Browser: chrome  

## Checklist:
